speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
> Nope it is not in my MOVIES slot, though (as we have discussed before) I am less convinced the contents of the slots matter that much.

They do matter in that it will have to fuzzy match less often.  Whether or not that works out better is another discussion, but it will in fact get exact an match earlier if it's in the slot.

> Maybe it would have worked if it was in the slot, but didn't need to be as it played fine when I said "play movie ghostbusters". Then when I added the articles to the sample utterances (still without ghostbusters in the slot) it then played fine when is said "play the movie ghostbusters"

I'm not trying to argue that you should populate your slots to get it to work.  I was just curious what the difference was between our deployments.  Obviously, if adding articles to the sample utterances makes it work for you, then that's the ultimate answer.  But for the sake of satisfying my own curiosity, I wouldn't mind pinpointing the exact difference.

> You could test at your end with a movie you know is not in your MOVIES slot, or any movie even if it is not in your library. Just to see what Alexa hears.

I'll do this tonight and report back.